### PR TITLE
David/split refactors from 1522

### PR DESCRIPTION
**Pull request by fedekunze**
_Description:_

Split from #1522 

❤️ Thank you!

---

* [x] Added entries in `CHANGELOG.md` with issue # and GitHub username
* [x] Reviewed `Files changed` in the github PR explorer

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -19,6 +19,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 * [\1401](https://github.com/cosmos/voyager/issues/1401) Display governance proposals index. @fedekunze
 * [\#1472](https://github.com/cosmos/voyager/issues/1472) Added mock functionality for redelegation @fedekunze + @faboweb
 * [\#1501](https://github.com/cosmos/voyager/issues/1501) Vote on proposals through modal @fedekunze + @jbibla
+* Various refactors on `main`, `node` and `lcdClient`. @NodeGuy
```
[the remark this answers is not in the document]
Comment (fedekunze) [reply CHANGELOG.md]: there's no issue, should I create one ?

[the remark this answers is not in the document]
Comment (fedekunze) [reply CHANGELOG.md]: Added

File: test/unit/specs/App.spec.js [hunk 2]
```diff
@@ -59,11 +57,11 @@ describe(`App without analytics`, () => {
     })
     let Node = require(`renderer/connectors/node.js`)
     require(`renderer/main.js`)
-    expect(Node).toHaveBeenCalledWith(
+    expect(Node.mock.calls[0].slice(1)).toEqual([
```
Comment (faboweb) [inline test/unit/specs/App.spec.js]: What was the problem here?

Comment (fedekunze) [reply test/unit/specs/App.spec.js]: @NodeGuy ^^

Comment (NodeGuy) [reply test/unit/specs/App.spec.js]: I added an initial argument `axios` to that function and `toHaveBeenCalledWith` was having trouble comparing functions so I opted to bypass it.

**Pull request merged**